<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataTableUIClass</class>
 <widget class="QWidget" name="DataTableUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataTableUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="ClearDataBtn">
         <property name="text">
          <string>Clear Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClearCountBtn">
         <property name="text">
          <string>Clear Count</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
